peas/weather.py

```diff
@@ -102,20 +98,10 @@ class AAGCloudSensor(object):
     """
 
     def __init__(self, serial_address=None, use_mongo=True):
-        self.config = load_config()
-        self.logger = logging.getLogger('aag-cloudsensor')
-        self.logger.setLevel(logging.INFO)
-
-        # Read configuration
-        self.cfg = self.config['weather']['aag_cloud']
-
-        self.safety_delay = self.cfg.get('safety_delay', 15.)
+        WeatherAbstract.__init__(self, use_mongo=True)
```

Comment:
Better to call `super().__init__(use_mongo=use_mongo)`.

The `super` is in place of an explicit named class and is just better practice. The `self` gets passed anyway, and the `use_mongo` should use the variable itself. Let me know if some of that doesn't make sense.

Reply:
I understand the why we would use `super` but why would we have `use_mongo=use_mongo`, not that I understood what it was doing before?

Reply:
In the `init` line you are declaring parameters, in this case a param `use_mongo` with a default value of `True`. Inside the method itself you want to use the actual parameter. It's a bit tricky here though because you are then passing that param to the `super()` and you want to pass that param into a param of the same name.

So if I say 

```
my_aag = AAGCloudSensor(use_mongo=False)
```

then inside the method itself the `use_mongo` will have a `False` value which gets passed to the `super()`. If you kept is `use_mongo=True` in the super call then it would always pass `True` rather than pass the param that the user creates like in example above.

Reply:
Is it because in the method we already defined `use_mongo=True`?

Reply:
Didn't see your reply before, that's what I thought but with more detail. Thanks!